docs/dasharo-tools-suite/documentation/features.md

```diff
@@ -313,17 +77,17 @@ And partially (only EC firmware flashing) on:
 * NovaCustom V540TU/TNx,
```

Comment:
@EduKav1813 let's move section related to HW `This feature is supported on the following platforms:` to the `supported-hardware.md` file. Then, let's link to the corresponding file.

Reply:
Resolved here: 499b1cf0d4e7b3b7150e604595cac99bd60d4000